regression/cbmc/reachability-slice-interproc2/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--reachability-slice --show-goto-functions
+^EXIT=0$
+^SIGNAL=0$
+main\(\)
+--
+^warning: ignoring
```

Comment:
Can you check that `b` is not in the slice?

Reply:
No, it is in the slice, since it calls c.

Reply:
Ah, I see, the forward walk is not used in the way I assumed.

Reply:
Is there a test that actually slices away something?

Reply:
The original cbmc/reachability-slice has two tests looking for slicing. These new ones were particularly looking for over-slicing since that's the bug @polgreen / @tautschnig encountered.